src/ContentRepository/Content.cs

```diff
@@ -749,7 +747,7 @@ private static ContentType ExtendContentType(ISupportsDynamicFields extendedHand
         /// Fully qualified type name is contained by the <see cref="SenseNet.ContentRepository.Schema.ContentType">ContentType</see> named this parameter's value.
         /// </param>
         /// <param name="parent"><see cref="SenseNet.ContentRepository.Storage.Node">ContentHandler</see> instance as a parent in the big tree.</param>

```

Comment:
contentTypeName parameter comment is missing

Reply:
See line 746 (above "parent" param).